### PR TITLE
Add "are you an exporter" check

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-33)

image

# "No intention" selected

image

# 'Nothing' selected
image

### CODE REVIEW

File: enrolment/forms.py [hunk 1]
```diff
@@ -24,6 +24,21 @@ class CompanyNameForm(forms.Form):
...
 
+class CompanyExportStatusForm(forms.Form):
+    export_status = forms.ChoiceField(
+        label=(
+            'Has your company sold products or services to overseas customers?'
+        ),
+        choices=constants.EXPORT_STATUSES,
+    )
+
+    def clean_export_status(self):
```
Comment (monikasulik) [inline enrolment/forms.py]: Shouldn't this be a validator in our validators library?

Comment (richtier) [inline enrolment/forms.py]: oh yes!

File: enrolment/templates/export-status-form.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{% extends "form-wrapper.html" %}
+{% load no_export_intention from enrolment_tags %}
+{% block form_title %}Join the UK Exporter Directory{% endblock%}
+{% block formarea_container_class %}
+    {% if form|no_export_intention %}
```
Comment (richtier) [inline enrolment/templates/export-status-form.html]: @konradko much better :)

**Comment by richtier** [comment]
@monikasulik the code now imports the error string from `export-validators`, so solving that problem :)

**Pull request merged**